### PR TITLE
Pop up

### PR DESCRIPTION
**Pull request by Stanleeeeee**
I used a javascript objects in creating the portfolio pop-up functionality. Whenever a see project is clicked a pop-up details shows.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -398,6 +441,258 @@ a {
   background-position-y: 380px;
 }
 
+/* Projects section */
+
+.popup {
+  width: 85vw;
+  height: 92vh;
+  padding: 1.8rem 1.2rem 1rem 1.2rem;
```
Comment (RolandM99) [inline style.css]: - [ ] Make sure that your design fits correctly the original design from Figma for the mobile version of your website.
### Original design from Figma:
image 
### Your Design
 image

- Kindly, fix this issue

**Review by RolandM99 — CHANGES_REQUESTED** [review]
Hi @Stanleeeeee  , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### To highlight 💯 
- Good commit message ✔️ 
- No Linter errors ✔️ 
- Follow Github flow ✔️ 

## Required Changes ♻️
_Check the comments under the review._

- [ ] Good job so far by implementing the pop-up window. But, there is an issue you can still work on to make your project great. Make sure that once the user refreshes or clicks on any button(menu or popup window) this error below doesn't appear in the console of your browser.
image

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on @Rolandm99 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Stanleeeeee — COMMENTED** [review]
This is the corrections for Responsiveness of the popup window of the portfolio project

**Review by topeogunleye — APPROVED** [review]
### Hi @Stanleeeeee, 

# Status: Approved ✔️
Excellent job on this project. Keep it up 👏
congratulations
Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**